src/routes/account.js

```diff
@@ -2,11 +2,17 @@
 import express from "express";
 import validate from "../middleware/validate.js";
 import schema from "../validation-schema/index.js";
-import { createAccount } from "../controllers/account.js";
+import { createAccount, getAccount, updateAccount, updateAccountWithId } from "../controllers/account.js";
 
 const router = express.Router();
-const { CREATE } = schema;
+const { CREATE, UPDATE } = schema;
 
 router.post("/", validate(CREATE), createAccount);
 
+router.get("/:id", getAccount);
+
+router.patch("/", validate(UPDATE), updateAccount);
+
+router.patch("/:id", updateAccountWithId);
```

Comment:
I think the validator middle-ware needs to be mounted here too
```suggestion
router.patch("/:id", validate(UPDATE), updateAccountWithId);
```

Reply:
The validation requires account number 
but if the id is passed in the url is there a need for account number? @jirevwe